cabal.project

```diff
@@ -15,6 +15,7 @@ packages:
     trace-dispatcher
     trace-resources
     trace-forward
+    cardano-tracer
```

Comment:
Sorry for the nitpicking, but I imagine people prefer to keep this list sorted.

Reply:
Fixed